src/chrome/content/rules/Storebot.me.xml

```diff
@@ -0,0 +1,17 @@
+<!--
+	Ruleset for storebot.me.
+
+	Not supporting HTTPS or using an invalid certificate:
```

Comment:
Could you be more verbose about the error (e.g. mismacth, slef-signed, https refused...)?